ui/shared/utils/constants.js

```diff
@@ -609,11 +609,6 @@ const VEP_SV_CONSEQUENCES = [
     text: 'Loss of function',
     value: 'LOF',
   },
-  {
```

Comment:
you should put `INTRAGENIC_EXON_DUP` over here

Reply:
Okay